src/protagonist/API/Features/Image/ImageController.cs

```diff
@@ -194,8 +214,9 @@ public async Task<IActionResult> GetImage(int customerId, int spaceId, string im
     }
 
     /// <summary>
-    /// Ingest specified file bytes to DLCS. Only "I" family assets are accepted.
-    /// "File" property should be base64 encoded image. 
+    /// <para>Ingest specified file bytes to DLCS. Only "I" family assets are accepted.

```

Comment:
What restricts this ti `I` only? (I know you've not changed that element of it but I can't see anything obvious from review).